lib/cocoapods/installer/user_project_integrator/target_integrator.rb

```diff
@@ -183,7 +189,11 @@ def add_check_manifest_lock_script_phase
                   echo "error: The sandbox is not in sync with the Podfile.lock. Run 'pod install' or update your CocoaPods installation." >&2
                   exit 1
               fi
+              # This output is used by Xcode 'outputs' to avoid re-running this script phase.
+              echo "SUCCESS" > "${SCRIPT_OUTPUT_FILE_0}"
```

Comment:
@segiddins it *does* work when the script contents are part of the Xcode. If the Xcode script just executes the `.sh` file then Xcode has no idea if the `.sh` file itself changed.

I am going to add for the `embed frameworks script` the `.sh` file itself as an `input`